meeteval/der/md_eval.py

```diff
@@ -0,0 +1,175 @@
+import logging
+import re
+import decimal
+import shutil
+import tempfile
+import subprocess
+import dataclasses
+from pathlib import Path
+
+import meeteval.io
+from meeteval.wer.wer.error_rate import ErrorRate
+
+
+def _fix_channel(r):
+    return meeteval.io.rttm.RTTM([
+        line.replace(channel='1')  # Thilo puts there usually some random value, e.g. <NA> for hyp and 0 for ref, while common default is 1
+        for line in r
+    ])
+
+
+@dataclasses.dataclass(frozen=True)
+class DiaErrorRate:
+    """
+
+    """
+    error_rate: 'float | decimal.Decimal'
```

Comment:
Why is this not computed from the times? The WER `ErrorRate` has `dataclasses.field(init=False)` and always computes it in the `__post_init__`

Reply:
While the WERs are calculated from our code, this is the values that is computed by md_eval_22.pl.
I wanted to be as close as possible to the original. (except, that it is here just the ratio and not the ratio in percent)